src/scripts/install-docker-compose.sh

```diff
@@ -33,38 +33,22 @@ if command -v docker-compose &> /dev/null; then
   fi
 fi
 
-# get binary/shasum download URL for specified version
+# get binary download URL for specified version
 if uname -a | grep Darwin &> /dev/null; then
   PLATFORM=darwin
   HOMEBREW_NO_AUTO_UPDATE=1 brew install coreutils
 else
   PLATFORM=linux
 fi
 
+FILENAME="docker-compose-$PLATFORM-x86_64"
 DOCKER_COMPOSE_BASE_URL="https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_VERSION"
-DOCKER_COMPOSE_SHASUM_URL="$DOCKER_COMPOSE_BASE_URL/docker-compose-$PLATFORM-x86_64.sha256"
```

Comment:
They stopped generating the `.sha256` files, but they've had a `checksums.txt` file for a number of releases now. So maybe if fetching the sha256 file fails, fall back to fetching checksums.txt and using that?